libopenage/datastructure/tests.cpp

```diff
@@ -192,4 +191,57 @@ void constexpr_map() {
 	TESTEQUALS(cmap.get(42), 9001);
 }
 
-}}} // openage::datastructure::tests
+std::mutex CopyMove::s_mutex = std::mutex();
+int CopyMove::s_accumulatedConstructionCounter = 0;
```

Comment:
I think you can define the classes in the `cpp` file, since they're used nowhere else except this test.

Reply:
Fixed. Thanks.